src/dsp_tools/utils/project_create.py

```diff
@@ -6,72 +6,142 @@
 
 from dsp_tools.models.connection import Connection
 from dsp_tools.models.group import Group
-from dsp_tools.models.helpers import BaseError, Cardinality, Context
+from dsp_tools.models.helpers import BaseError, Cardinality, Context, DateTimeStamp
 from dsp_tools.models.langstring import LangString
 from dsp_tools.models.ontology import Ontology
 from dsp_tools.models.project import Project
 from dsp_tools.models.propertyclass import PropertyClass
 from dsp_tools.models.resourceclass import ResourceClass
 from dsp_tools.models.user import User
 from dsp_tools.utils.excel_to_json_lists import expand_lists_from_excel
-from dsp_tools.utils.onto_create_lists import create_lists
-from dsp_tools.utils.onto_validate import validate_project
+from dsp_tools.utils.project_create_lists import create_lists
+from dsp_tools.utils.project_validate import validate_project
 from dsp_tools.utils.shared import login, try_network_action
 
 
-def _create_project(con: Connection, project_definition: dict[str, Any]) -> Project:
+def _create_project(
+        input_file: str,
+        server: str,
+        user_mail: str,
+        password: str,
+        verbose: bool,
+        dump: bool
+) -> Tuple[Connection, Context, dict[str, Any], Project, bool]:
```

Comment:
these kind of return types normally indicate that you might want to consider using a DTO (data transfer object), which could be a simple dataclass like this

```python
@dataclass(frozen=True)
class createProjectResponse:  # or come up with a better name
    connection: Connection
    context: Context
    json: dict[str, Any]
    project: Project
    success: bool
```

so, essentially it's really just a named tuple, but it's far more expressive than a simple Tuple.  
For now you can absolutely leave it like this. But in the future, it may be something to consider

Reply:
cool, I note that for later